### PR TITLE
change secret types example

### PR DESCRIPTION
**Pull request by ashears**
## Change Summary
I was examining https://github.com/samuelcolvin/pydantic/pull/884, which discussed the secret types. I found that the example for secret types could be slightly more clear.

Apologies if I am in error with creating this PR, I do not have experience submitting PR to an open source code base.

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)

### CODE REVIEW

**Comment by dmontagu** [comment]
Looks good to me

> changes/<pull request or issue id>-<github username>.md file added describing change
(see changes/README.md for details)

Feel free to do this if you want credit for your contribution!

File: docs/examples/ex_secret_types.py [hunk 1]
```diff
@@ -7,17 +7,20 @@ class SimpleModel(BaseModel):
     password_bytes: SecretBytes
 
 sm = SimpleModel(password='IAmSensitive', password_bytes=b'IAmSensitiveBytes')
+
+# Standard access methods will not display the secret
 print(sm)
-#> SimpleModel password=SecretStr('**********') password_bytes=SecretBytes(b'**********')
+# > SimpleModel password=SecretStr('**********') password_bytes=SecretBytes(b'**********')
```
Comment (samuelcolvin) [inline docs/examples/ex_secret_types.py]:
```suggestion
#> SimpleModel password=SecretStr('**********') password_bytes=SecretBytes(b'**********')
```
and below.

File: docs/examples/ex_secret_types.py [hunk 2]
```diff
@@ -7,17 +7,20 @@ class SimpleModel(BaseModel):
...
+print(sm.password.display())
```
Comment (samuelcolvin) [inline docs/examples/ex_secret_types.py]: `.display()` is deprecated after #884 which will be merged before this (try to merge in the order PRs are created unless there's a good reason not to)

Comment (ashears) [reply docs/examples/ex_secret_types.py]: Changed to `print(str(sm.password))`

File: docs/examples/ex_secret_types.py [hunk 3]
```diff
@@ -5,12 +5,15 @@ class SimpleModel(BaseModel):
     password_bytes: SecretBytes
 
 sm = SimpleModel(password='IAmSensitive', password_bytes=b'IAmSensitiveBytes')
+
+# Standard access methods will not display the secret
 print(sm)
+print(str(sm.password))
```
Comment (samuelcolvin) [inline docs/examples/ex_secret_types.py]:
```suggestion
print(repr(sm.password))
```
`print(str(x))` is the same as `print(x)`

**Comment by samuelcolvin** [comment]
thank you very much.

**Pull request merged**